### DISCUSSION TITLE
Hactoberfest Participation

### DISCUSSION THREAD

**Discussion by SidharthMudgil** [Midway91/HactoberFest2023#13] [General]
Please add `hactoberfest` tag in the repository otherwise it will not be qualified. `hactoberfest2023` is not a valid tag

**Comment by AnushkJain2201** [comment]
Yes please add the tag hactoberfest

**Comment by AnushkJain2201** [comment]
Also add label hactoberfest-accepted while merging our PRs

**Reply by Midway91** [reply]
Would you be willing to put some screen shots in here of how to do this? Thanks.

**Comment by Midway91** [comment]
@SidharthMudgil @AnushkJain2201 
I will do that let me know if I am doing anything else incorrectly because it is my first year thanks.

**Reply by AnushkJain2201** [reply]
Yes we will let you know

**Comment by AnushkJain2201** [comment]
Just add the tag hactoberfest-accepted when you merge any PR.
OR message me on 7095830205 wa

**Comment by SidharthMudgil** [comment]
No need for hactoberfest fest accepted just add hactoberfest tag

**Reply by Midway91** [reply]
@SidharthMudgil 
To the individual pr or just to the whole repository

**Reply by SidharthMudgil** [reply]
Whole repo

**Reply by SidharthMudgil** [reply]
If pr is merged it will automatically treated as accepted

**Reply by SidharthMudgil** [reply]
To be accepted, a maintainer either needs to merge your PR/MR, add the hacktoberfest-accepted label, or leave an overall approving review.

For more information about how exactly we validate PR/MRs for contributors, check the details on our participation page.

**Comment by AnushkJain2201** [comment]
hacktoberfest*

**Comment by SidharthMudgil** [comment]
👍
